arcade/scene.py

```diff
@@ -22,9 +22,13 @@ class Scene:
     Class that represents a `scene` object. Most games will use Scenes to render their Sprites.
     For examples on how to use this class, see:
     https://api.arcade.academy/en/latest/tutorials/views/index.html
-
     """
 
+    __slots__ = (
+        "_sprite_lists",
+        "_name_mapping",
+    )
+
```

Comment:
I'm strongly opposed to including these on `Scene` because it may create serious downsides. Imo, `Scene` is a strong candidate for subclassing to customize behavior, ie including other drawables such as the background layers which DragonMoffon and I need to do API cleanup on.

Reply:
I agree, had the same feeling. @gran4 could you remove slots from Scene?